app/build/plugins/image/index.js

```diff
@@ -11,43 +13,58 @@ function render ($, callback, options) {
   // Process 5 images concurrently
   eachEl($, 'img', function(el, next){
 
-    var src = $(el).attr('src');
+    // Decode any doubly-encoded ampersands in the image src
+    var src = decodeAmpersands($(el).attr('src'));
     var width, height;
 
-    debug(src, 'checking cache');
+    // Test for query string to skip caching & optimizing
+    var parsedSrc = url.parse(src, { parseQueryString: true });
+    debug(src, 'is parsed into', parsedSrc)
 
-    cache.lookup(src, optimize(blogID), function(err, info){
+    if (parsedSrc.query.static) {
```

Comment:
Lines 24 and 21 should probably be inside a try/catch since we're dealing with user input and URL.parse throws errors [under certain conditions](https://nodejs.org/api/url.html#url_url_parse_urlstring_parsequerystring_slashesdenotehost).

Reply:
Yeah, looks like it's either a `TypeError` because it's not a string or a `URIError` because of a mangled/malformed `auth` field that can't be decoded.

Seems like it would make sense to wrap those few lines in the `try` part & just bail out in the `catch` with a `return next()`?

Reply:
Sounds like a good plan, will make these modifications